### PR TITLE
VY-COM-13 - Add role field in User DTO

### PR DESCRIPTION
**Pull request by lf-achyutpkl**

### CODE REVIEW

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

File: src/main/java/com/lftechnology/vyaguta/commons/pojo/Role.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ *
+ */
+public class Role {
+    private String id;
+
+    private String role;
+
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
+    @Convert(converter = LocalDateTimeAttributeConverter.class)
```
Comment (lf-kailashraj) [inline src/main/java/com/lftechnology/vyaguta/commons/pojo/Role.java]: remove converter

**Comment by Ankita01** [comment]
Why the hell i am receiving mails from you guys!
Please remove my id from this chain.
Id- ankita.gupta0104@gmail.com

If i get another mail, i'll report you as spam

**Pull request merged**